### PR TITLE
Add support for subscription events

### PR DESCRIPTION
**Pull request by sbrych**

### CODE REVIEW

**Review by azranel — APPROVED** [review]

File: lib/chartmogul/subscription_event.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  class SubscriptionEvent < APIResource
+    set_resource_name 'SubscriptionEvent'
+    set_resource_path '/v1/subscription_events'
+
+    readonly_attr :id
+    writeable_attr :data_source_uuid
```
Comment (pkopac) [inline lib/chartmogul/subscription_event.rb:9]: [nitpick] Could you please apply some sorting?

File: spec/chartmogul/subscription_event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ChartMogul::SubscriptionEvent do
+  describe 'API interactions', vcr: true do
+    it 'lists all subscription events', uses_api: true do
+      data_source = ChartMogul::DataSource.new(
```
Comment (pkopac) [inline spec/chartmogul/subscription_event_spec.rb:8]: [duplication] This whole test setup is repeated and should go into a `before` block, please.

**Review by pkopac — APPROVED** [review]
It looks fine... I feel like I am doing CodeClimate's job here, though... why isn't it submitting a report? :thinking:

File: spec/chartmogul/subscription_event_spec.rb (hunk 2 above)
Comment (sbrych) [reply spec/chartmogul/subscription_event_spec.rb:8]: Will do in a separate PR

**Pull request merged**